Firestore/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# v7.8.0
```

Comment:
Next release is 7.9.0. 7.8.0 already released.

Reply:
Done.